test/e2e/fixture/zomes/foo/Cargo.toml

```diff
@@ -10,4 +10,4 @@ crate-type = [ "cdylib", "rlib" ]
 
 [dependencies]
 serde = "=1.0.123"
-hdk = {git ="https://github.com/holochain/holochain", rev = "c5dbdf28825927106bc32d186dd54f20d35df468", package = "hdk"}
+hdk = "0.0.106"
```

Comment:
```suggestion
hdk = "=0.0.106"
```

Reply:
Else it will get overridden when we have a new release. Unless that's what we want